captslog/ui/mainwindow.py

```diff
@@ -25,34 +17,39 @@ def _translate(context, text, disambig):
         return QtGui.QApplication.translate(context, text, disambig)
 
 
-# Main Window GUI
 class Ui_MainWindow(QtGui.QMainWindow):
 
     def setupUi(self, MainWindow):
-        """Setting up Main window UI.
-        """
-        self.setObjectName(_fromUtf8("MainWindow"))
-        self.resize(550, 600)
-
-        # Size Policy
-        sizePolicy = QtGui.QSizePolicy(
-            QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding)
-        sizePolicy.setHorizontalStretch(0)
-        sizePolicy.setVerticalStretch(0)
-        sizePolicy.setHeightForWidth(
-            self.sizePolicy().hasHeightForWidth())
+		self.setObjectName(_fromUtf8("MainWindow"))
+		
+		"""Set up the size of the Window
```

Comment:
There shouldn't be a documentation here.
Documentations are for functions and classes.